templates/restic_script_Linux.j2

```diff
@@ -102,6 +102,39 @@ if [[ -z ${CRON+x} ]]; then
 else
   MODE_TAG="--tag cron"
 fi
+
+
+{% if item.enable_logging is defined and item.enable_logging %}
```

Comment:
Enable logging should render the appending of the output to a file, not rerender the entire script